README.md

```diff
@@ -231,6 +231,24 @@ The device plugin will initially discover all PCI network resources in the host
 3. "drivers" - The driver name the device is registered with
 4. "pfNames" - The Physical funtion name
 
+To change the NIC name using the udev rules:
```

Comment:
Might be worth mentioning why we are adding udev rule settings here, for example, it's for better use of the `pfNames` filter across cluster nodes, to configure identical interface names for devices on different nodes.